src/components/views/rooms/RoomHeader.tsx

```
@@ -85,7 +85,15 @@ import { Alignment } from "../elements/Tooltip";
 import RoomCallBanner from "../beacon/RoomCallBanner";
 import { shouldShowComponent } from "../../../customisations/helpers/UIComponents";
 import { UIComponent } from "../../../settings/UIFeature";
-import { getTopic } from "../../../hooks/room/useTopic";
+
+
+
+
+
+import ResizeNotifier from "../../../utils/ResizeNotifier";
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)